doc/source/ray-air/examples/tfx_tabular_train_to_serve.ipynb

```diff
@@ -0,0 +1,2059 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VaFMt6AIhYbK"
+   },
+   "source": [
+    "This notebook is adapted from \"components keras\" [tutorial](https://www.tensorflow.org/tfx/tutorials/tfx/components_keras).\n",
+    "The goal is to predict whether a trip may generate a big tip.\n",
+    "\n",
+    "In this example, we showcase how to convert a tfx pipeline to [Ray AIR](https://docs.ray.io/en/latest/ray-air/getting-started.html), covering\n",
+    "every step from data ingestion to pushing a model to serving.\n",
+    "\n",
+    "1. Read a CSV file into ray dataset.\n",
```

Comment:
Here are the steps we will take below:

1. Read a CSV into Ray Dataset (link)
2. Process the dataset by chaining Ray AIR preprocessors (link)
3. Train the model using the TensorflowTrainer from AIR
4. Serve the model using Ray Serve and the above preprocessors